vsts/pipelines/templates/_buildTemplate.yml

```diff
@@ -59,6 +59,13 @@ steps:
   inputs:
     ignoreDirectories: '$(Build.SourcesDirectory)/tests'
 
+- task: Docker@1
+  displayName: Container registry logout
+  inputs:
+    command: logout
+    azureSubscriptionEndpoint: ${{ parameters.ascName }}
+    azureContainerRegistry: ${{ parameters.acrName }}
+
```

Comment:
why did we add this? looks like we are logging in as the next step

Reply:
Ideally login should have worked but it was failing with new service connection.
I have seen in other pipeline where it caches the data.. so i thought to logout and login but still it did not work.
So removing this.